src/Features/Core/Portable/ExtractMethod/ExtractMethodResult.cs

```diff
@@ -64,5 +75,25 @@ internal class ExtractMethodResult
         /// internal status of result. more fine grained reason why it is failed. 
         /// </summary>
         internal OperationStatusFlag Status { get; }
+
+        public async Task<(Document document, SyntaxToken invocationNameToken)> GetFormattedDocumentAsync(CancellationToken cancellationToken)
+        {
+            if (DocumentWithoutFinalFormatting is null)
+                throw new InvalidOperationException();
+
+            var annotation = new SyntaxAnnotation();
```

Comment:
consider a static.  it's our general pattern.  up to you.

Reply:
➡️ Extract method doesn't use static annotations for this.

https://github.com/dotnet/roslyn/blob/c7521c9ec5dbdc06778606960fe211c26420d826/src/Features/Core/Portable/ExtractMethod/MethodExtractor.CodeGenerator.cs#L55-L57

This one seems fine for such a localized use.